scripts/boot-master/install-docker.sh

```diff
@@ -70,8 +70,9 @@ function ubuntu_docker_install {
   curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
 
   # Right now hard code adding x86 repo
+  local arch=$(uname -m | sed 's/x86_64/amd64/g')
   sudo add-apt-repository \
-   "deb [arch=amd64] https://download.docker.com/linux/ubuntu \
+   "deb [arch=${arch}] https://download.docker.com/linux/ubuntu \
```

Comment:
This will not work for ppc64le, the `arch` needs to be `ppc64el` according to https://docs.docker.com/install/linux/docker-ce/ubuntu/#install-docker-ce.

Reply:
I have updated the PR, it can be support ppc64el